tools/prlint/module.ts

```diff
@@ -0,0 +1,37 @@
+import * as fs from 'fs-extra';
+import * as glob from 'glob';
+import * as path from 'path';
+
+const modules: string[] = [];
+
+export function findModulePath(fuzz: string): string {
+  if (modules.length === 0) {
```

Comment:
can we extract this `findAllModules` into a separate function that will be called once (outside of the `map`) in `assertStability`?

Reply:
Simply for code cleanliness and avoiding a global variable